packages/expo-image/src/Image.types.ts

```diff
@@ -178,6 +178,18 @@ export interface ImageProps extends Omit<ViewProps, 'style' | 'children'> {
    */
   priority?: 'low' | 'normal' | 'high' | null;
 
+  /**
+   * Sets the HTML `loading` attribute on the `<img>` element.
+   * Has no effect on native platforms.
+   *
+   * - `'lazy'` - Defers loading until the image is near the viewport.
+   * - `'eager'` - Loads the image immediately.
+   *
+   * @default undefined
```

Comment:
Isn't `eager` the default?
```suggestion
   * @default 'eager'
```